src/Header.qml

```diff
@@ -24,6 +24,8 @@ SilicaFlickable {
 
     property alias searchFocus: placeHeader.searchFocus
     property alias searchText: placeHeader.text
+    property alias searching: placeHeader.searching
```

Comment:
Maybe don't expose these two internals as suggest in the comment in the main.qml startSearch() function.